src/app/space/create/deployments/resource-usage/deployments-resource-usage.component.ts

```diff
@@ -10,10 +10,7 @@ import { Environment } from '../models/environment';
   templateUrl: 'deployments-resource-usage.component.html'
 })
 export class DeploymentsResourceUsageComponent {
-
   @Input() environments: Observable<Environment[]>;
   @Input() spaceId: Observable<string>;
-
-  constructor() { }
-
+  resourcesCollapsed: boolean = false;
```

Comment:
I'd prefer a newline between annotated variables and ones that aren't annotated.